ddtrace/settings/__init__.py

```diff
@@ -1,12 +1,10 @@
 from .._hooks import Hooks
-from ._config import Config
 from .exceptions import ConfigException
 from .http import HttpConfig
 from .integration import IntegrationConfig
 
 
 __all__ = [
-    "Config",
```

Comment:
Can we avoid this? It feels like a breaking change. If this removal resolves a circular conflict, then I am fine with this change

Reply:
This was removed to fix the circular imports. I don't think this would break anything as nothing else other than the internals need to create objects of this type